hack/make-rules/test-e2e-node.sh

```diff
@@ -47,6 +47,7 @@ ssh_user=${SSH_USER:-"${USER}"}
 ssh_key=${SSH_KEY:-}
 ssh_options=${SSH_OPTIONS:-}
 kubelet_config_file=${KUBELET_CONFIG_FILE:-"test/e2e_node/jenkins/default-kubelet-config.yaml"}
+instance_type=${INSTANCE_TYPE:-}
```

Comment:
Please add the "t3a.medium" as default here.

Reply:
The runner.go sets the default now.